src/App.js

```diff
@@ -1,25 +1,50 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useEffect, useState } from 'react';
+import GameBoard from './Components/Board/Board';
+import _ from 'lodash';
+import Cell from './Components/Cell/Cell';
 import './App.css';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+  const gameWidth = 3;
+  const gameHeight = 3;
+  let [turn,setTurn]=useState(0);
```

Comment:
use consts and only lets when you need to

Reply:
Do you mean this in the sense of let **[turn,setTurn]=useState(0)**; I thought since turn is always going to be changed with by the setTurn method, it should not be a constant. 
I may be totally off though.

Reply:
Pretty much everywhere you've got let, you can use const.

Yeah, just read up the difference between const and let